src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java

```diff
@@ -433,8 +430,6 @@ public void setFetchDirection(int direction) throws SQLException {
 	}
 
 	@Override
-	@NotImplemented
-	@ExcludeFromJacocoGeneratedReport
 	public int getFetchSize() throws SQLException {
 		return 0;
 	}
```

Comment:
This should stay "not implemented". From Oracle's docs, this seems to be the pagination size so 0 would be misleading. https://docs.oracle.com/cd/E11882_01/java.112/e16548/resltset.htm#JJDBC28621

Reply:
restored `@NotImplemented`